packages/vuetify/src/components/VDivider/VDivider.sass

```diff
@@ -1,39 +1,32 @@
 @import './_variables.scss'
 
-+theme(v-divider) using ($material)
-  border-color: map-get($material, 'dividers')
-
 .v-divider
+  border-style: $divider-border-style
+  border-width: $divider-border-width
   display: block
-  flex: 1 1 0px
```

Comment:
This was here to fix #4641

Reply:
I checked it and it's no longer needed in v2.4, so no point in bringing it to v3.

Reply:
Changing this to 100% so these work in flex containers. The only other way I could get it to work was using `min-width: 1px` which felt wrong.